### PR TITLE
Added MessageBoard and message_board_spec

### PR DESCRIPTION
**Pull request by adamxlowe**

### CODE REVIEW

**Comment by jwo** [comment]
If I run these specs, it fails:

```
  1) Conductor should tell the engineer to slow down
     Failure/Error: let(:engineer) { Engineer.new }
     ArgumentError:
       wrong number of arguments (0 for 1)
     # ./train.rb:16:in `initialize'
     # ./train_spec.rb:5:in `new'
     # ./train_spec.rb:5:in `block (2 levels) in <top (required)>'
     # ./train_spec.rb:9:in `block (2 levels) in <top (required)>'
```

Do they fail for you?

**Comment by adamxlowe** [comment]
I added a message board spec for the changed code. I don't know why it
didn't push before.

require 'rspec'
require './train'
describe MessageBoard do

  let(:message_board) { MessageBoard.new }
    let(:engineer) { Engineer.new(message_board) }
    let(:conductor) { Conductor.new(message_board)}

```
it "should tell the engineer to slow down" do
    message_board.should_receive(:slow_down!)
    conductor.see_danger_coming!
end
```

  it "should confirm to the conductor that it slowed down" do
    message_board.should_receive(:confirm_slow_down)
    engineer.slowing_down!
  end
end

**Comment by jwo** [comment]
so, the specs are still failing for me. Are they passing for you?

**Comment by adamxlowe** [comment]
rspec message_board_spec.rb

..

Top 2 slowest examples (0.00072 seconds, 100.0% of total time):

  MessageBoard should tell the engineer to slow down

```
0.00045 seconds ./message_board_spec.rb:9
```

  MessageBoard should confirm to the conductor that it slowed down

```
0.00027 seconds ./message_board_spec.rb:14
```

Finished in 0.00123 seconds

2 examples, 0 failures

Randomized with seed 6127

**Comment by jwo** [comment]
Hmm, but there are more spec files than the message_board_spec, right?

**Comment by adamxlowe** [comment]
That's the spec I wrote for the assignment.

**Comment by jwo** [comment]
Hey Adam --

You should be running `rspec .` --- and if the spec files were in a `spec`
directory, you should run `rspec spec`

tl;dr: You have a failing spec, and should not.

On Tue, Mar 4, 2014 at 2:44 PM, Adam Lowe notifications@github.com wrote:

> That's the spec I wrote for the assignment.
> 
> On Tue, Mar 4, 2014 at 11:42 AM, Jesse Wolgamott
> 
> notifications@github.comwrote:
> 
> > Hmm, but there are more spec files than the message_board_spec, right?
> > 
> > On Tue, Mar 4, 2014 at 2:38 PM, Adam Lowe notifications@github.com
> > wrote:
> > 
> > > rspec message_board_spec.rb
> > > ..
> > > Top 2 slowest examples (0.00072 seconds, 100.0% of total time):
> > > MessageBoard should tell the engineer to slow down
> > > 0.00045 seconds ./message_board_spec.rb:9
> > > MessageBoard should confirm to the conductor that it slowed down
> > > 0.00027 seconds ./message_board_spec.rb:14
> > > Finished in 0.00123 seconds
> > > 2 examples, 0 failures
> > > Randomized with seed 6127
> > > On Tue, Mar 4, 2014 at 11:32 AM, Jesse Wolgamott
> > > notifications@github.comwrote:
> > > 
> > > > so, the specs are still failing for me. Are they passing for you?
> > > > 
> > > > ## 
> > > > 
> > > > Reply to this email directly or view it on GitHub<
> > 
> > https://github.com/RubyoffRails/train-message_expectations/pull/11#issuecomment-36664000
> > 
> > > > .
> > > > 
> > > > ---
> > > > 
> > > > Reply to this email directly or view it on GitHub:
> > 
> > https://github.com/RubyoffRails/train-message_expectations/pull/11#issuecomment-36664641
> > 
> > ## 
> > 
> > Reply to this email directly or view it on GitHub<
> > https://github.com/RubyoffRails/train-message_expectations/pull/11#issuecomment-36665076
> > 
> > .
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/RubyoffRails/train-message_expectations/pull/11#issuecomment-36665314
> .

## 

# 

Jesse Wolgamott @ Comal Productions
Web. Mobile. Training. Code.

http://comalproductions.com
Twitter: @jwo
713.234.0361
